nixos/modules/programs/wayland/waybar.nix

```diff
@@ -13,11 +13,9 @@ in
 
   config = mkIf cfg.enable {
     environment.systemPackages = [ cfg.package ];
-    systemd.user.services.waybar = {
-      description = "Waybar as systemd service";
-      wantedBy = [ "graphical-session.target" ];
-      partOf = [ "graphical-session.target" ];
-      script = "${cfg.package}/bin/waybar";
+    systemd = {
+      packages = [ cfg.package ];
+      user.services.waybar.wantedBy = [ "graphical-session.target" ];
```

Comment:
Isn't that already here https://github.com/Alexays/Waybar/blob/master/resources/waybar.service.in#L13-L14 ?

Reply:
Yes but the Install section is not respected with `systemd.packages` (#81138)